platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -43,17 +43,21 @@ extension PolyOut {
         return documentDirectory.appendingPathComponent(PolyOut.fsFilesRoot).appendingPathComponent(activeFeature!.id)
     }
     
-    private func pathFromUrl(url: String) -> URL? {
-        let id = idFromUrl(url: url)
-
-        return id != nil ? featureFilesPath().appendingPathComponent(id!) : nil
-    }
+    /// There are 2 types of urls that are used
+    /// 1. Filesystem url that starts with file:// and points to a physical location on the device
+    /// 2. Polypod url that starts with polypod:// and is how features access their resources in a platform independent way
     
-    func pathFromId(id: String) -> URL {
+    func filesystemUrlFromId(id: String) -> URL {
         return featureFilesPath().appendingPathComponent(id)
     }
     
-    private func idFromUrl(url: String) -> String? {
+    private func filesystemUrlFromPodUrl(url: String) -> URL? {
+        let id = idFromPodUrl(url: url)
+
+        return id != nil ? featureFilesPath().appendingPathComponent(id!) : nil
```

Comment:
Just a suggestion, this could be simplified to something like this:
```suggestion
       idFromPodUrl(url: url).map(filesystemUrlFromId(id:))
```